### DISCUSSION TITLE
[question]您好，请问如何给图上节点添加点击事件呢？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/X6#4388] [Q&A]
在看到这个图的时候：
https://x6.antv.antgroup.com/zh/examples/showcase/practices#class

![20240813130402](https://github.com/user-attachments/assets/deb3b64a-693c-4591-9b62-8219dbe50a09)

请问下，如何给这上面的节点添加点击事件呢？

**Comment by zlecheng** [comment]
> 在看到这个图的时候： https://x6.antv.antgroup.com/zh/examples/showcase/practices#class
> 
> ![20240813130402](https://private-user-images.githubusercontent.com/12727582/357281668-deb3b64a-693c-4591-9b62-8219dbe50a09.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyMDIzMzAsIm5iZiI6MTcyNDIwMjAzMCwicGF0aCI6Ii8xMjcyNzU4Mi8zNTcyODE2NjgtZGViM2I2NGEtNjkzYy00NTkxLTliNjItODIxOWRiZTUwYTA5LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIxVDAxMDAzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThhMGVjMTY5Y2M5YmI5ZGJiYTVkYzM1NjFhNmM0ZDk5ODYxN2JkM2ZmZTQyY2U2MjMwMWI4NmE4YjJjZjdhMzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wbNklrWxBxQ-aUbaISbl7KFBKwn0kkxZ93-L1i5gJcs)
> 
> 请问下，如何给这上面的节点添加点击事件呢？

蓝色那块区域添加点击事件？

**Reply by liaodalin19903** [reply]
您好是的。
